local_build_instructions.md

```diff
@@ -0,0 +1,42 @@
+# Windows local build instructions
+
+Building bevy-website locally is tedious, especially on Windows. Here is one (brute force) method:
+
+1. [Install WSL](https://learn.microsoft.com/en-us/windows/wsl/install), with Ubuntu.
+
+note: You could instead use [git bash](https://gitforwindows.org/), but some of the details might be different.
+
+2. Open a WSL bash shell
+3. [install rustup](https://rustup.rs/) `curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh`
+4. Add the wasm target `rustup target add wasm32-unknown-unknown`
+5. Install build dependencies `sudo apt install build-essential pkg-config`
+6. Install wasm-bindgen tool `cargo install wasm-bindgen-cli`
+
+Now we have everything we need to start generating the site. We can roughly follow the
+outline of the CI build script `.github/workflows/ci.ympl`
+
+7. cd into the website repo. There are a bunch of directories starting with "generate-".
```

Comment:
I would add that these steps are optional if you don't plan to work on those related pages.

Reply:
Like, when I was working on the asset page, I never generated all the other pages.

Reply:
Thinking about it, we should probably just provide a `generate.sh` at the root that does all those steps

Reply:
Agree with you on git-bash being more minimal.. But I'm also not going to install a whole rust environment to git-bash when I already use WSL. I think there's value in having both.

I was hoping we could either flesh this out to include the other platforms (which will be pretty similar to the bulk, but I can't test mac), or just provide an automation for setting up the environment.